src/components/Header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Header.scss';
+
+function Header() {
+    return (
+        <header className="Header">
```

Comment:
The header does not need a header class because it's the only header component on the page.